Makefile

```diff
@@ -106,10 +106,20 @@ test_native: DIR=coverage_$(shell date -Is)
 test_native: ## run native tests (all tools and build dependencies are assumed to be available in the environment)
 	find FV3 -type f -name '*.gcda' -delete
 	pytest --native tests/pytest
+	pytest FV3/wrapper/tests/
 	mkdir -p $(DIR) && \
 		cd $(DIR)  && \
 		gcovr -d -r ../FV3 --html --html-details -o index.html
 
+test_wrapper:
+	$(MAKE) -C FV3/wrapper/ test
+
+lint:
```

Comment:
Do we normally have make targets for pre-commit install? I don't think we need it necessarily, just bringing it up in case.

Reply:
I added a setup-hooks target: a15e1c42609d87eae06778a660a5cb30cc7999c9